src/app/server/Server.h

```diff
@@ -522,6 +525,10 @@ class Server
     uint16_t mOperationalServicePort;
     uint16_t mUserDirectedCommissioningPort;
     Inet::InterfaceId mInterfaceId;
+
+#ifdef CHIP_CONFIG_USE_DEFAULT_READ_HANDLER_CALLBACKS
+    app::GenericReadHandlerCallback genericReadHandlerCallback;
```

Comment:
Please follow naming conventions (members start with `m` here).

Reply:
This can change the binary interface between binary libraries and integrations. Does it really need to be in Server class? Could this not be set by clients of the code by calling the `InteractionModelEngine::RegisterReadHandlerAppCallback` ? Having Server own this is otherwise not necessary

Reply:
done